geode-docs/tools_modules/geode_for_redis.html.md.erb

```diff
@@ -25,160 +25,320 @@ optional password authentication.
 
-<%=vars.product_name%> for Redis supports the following Redis commands.
-<br/>
-
- - APPEND <br/>
- - AUTH <br/>
- - DECR <br/>
- - DECRBY <br/>
- - DEL <br/>
- - EXISTS <br/>
- - EXPIRE <br/>
- - EXPIREAT <br/>
- - GET <br/>
- - GETRANGE <br/>
- - HDEL <br/>
- - HEXISTS <br/>
- - HGET <br/>
- - HGETALL <br/>
- - HINCRBY <br/>
- - HINCRBYFLOAT <br/>
- - HLEN <br/>
- - HMGET <br/>
- - HMSET <br/>
- - HSCAN **[1]**  <br/>
- - HSET <br/>
- - HSETNX <br/>
- - HSTRLEN <br/>
- - HVALS <br/>
- - HKEYS <br/>
- - INCR <br/>
- - INCRBY <br/>
- - INCRBYFLOAT <br/>
- - INFO **[2]**  <br/>
- - KEYS <br/>
- - MGET <br/>
- - PERSIST <br/>
- - PEXPIRE <br/>
- - PEXPIREAT <br/>
- - PING <br/>
- - PSUBSCRIBE <br/>
- - PTTL <br/>
- - PUBLISH <br/>
- - PUNSUBSCRIBE <br/>
- - QUIT <br/>
- - RENAME <br/>
- - SADD <br/>
- - SCARD <br/>
- - SDIFF <br/>
- - SDIFFSTORE <br/>
- - SINTER <br/>
- - SISMEMBER <br/>
- - SET <br/>
- - SETNX <br/>
- - SLOWLOG **[3]**  <br/>
- - SMEMBERS <br/>
- - SMOVE <br/>
- - SREM <br/>
- - STRLEN <br/>
- - SUBSCRIBE <br/>
- - SUNION <br/>
- - TTL <br/>
- - TYPE <br/>
- - UNSUBSCRIBE <br/>
-
-<br/>
 Commands not listed above are **not implemented**.
 
-<br/>
 **NOTES:**
 
 These commands are supported for Redis 5.
+**[1]]** CLUSTER is implemented for the subcommands INFO, NODES, SLOTS, and KEYSLOT.
 
-**[1]** Redis accepts 64-bit signed integers for the HSCAN cursor and COUNT parameters.
- <%=vars.product_name%> for Redis is limited to 32-bit integer values for these parameters.
+**[2]** COMMAND is implemented only with no subcommands.
 
-**[2]** INFO is implemented for the sections and fields listed below:
+**[3]** Redis accepts 64-bit signed integers for the HSCAN cursor and COUNT parameters.
+<%=vars.product_name%> for Redis is limited to 32-bit integer values for these parameters.
 
- - clients
+**[4]** INFO is implemented for the sections and fields listed below:
 
-    - connected_clients
+- clients
 
-    - blocked_clients (always returns 0)
+   - connected_clients
 
- - cluster
+   - blocked_clients (always returns 0)
 
-    - cluster_enables (always returns 0)
+- cluster
 
- - keyspace
+   - cluster_enables (always returns 0)
 
-    - returns stats for db: 0
+- keyspace
 
- - memory
+   - returns stats for db: 0
 
-    - maxmemory
+- memory
 
-    - used_memory
+   - maxmemory
 
-    - mem_fragmentation_ratio (always reports 1.00)
+   - used_memory
 
- - persistence
+   - mem_fragmentation_ratio (always reports 1.00)
 
-    - loading (always returns 0)
+- persistence
 
-    - rdb_changes_since_last_save (always returns 0)
+   - loading (always returns 0)
 
-    - rdb_last_save_time (always returns 0)
+   - rdb_changes_since_last_save (always returns 0)
 
- - replication
+   - rdb_last_save_time (always returns 0)
 
-    - role
+- replication
 
-    - connected_slaves (always returns 0)
+   - role
 
- - server
+   - connected_slaves (always returns 0)
+
+- server
 
    - redis_version
```

Comment:
Below this, redis_mode always returns "cluster" because Geode for Redis always runs in cluster mode.